grammar.js

```diff
@@ -168,22 +168,31 @@ module.exports = grammar({
             choice(
                 // seq($.member, repeat(seq(",", $.member)), optional(",")),
                 commaSep1($.member, true),
-                seq(
-                    repeat(seq($.objlocal, ",")),
-                    "[",
-                    $.expr,
-                    "]",
-                    ":",
-                    $.expr,
-                    repeat(seq(",", $.objlocal)),
-                    optional(","),
-                    $.forspec,
-                    optional($.compspec)
-                )
+                $._objforloop,
+            ),
+
+        _objforloop: ($) =>
+            seq(
+                repeat(seq($.objlocal, ",")),
+                //"[", $.expr, "]",
+                //":",
+                //$.expr,
+                $.field,
```

Comment:
This is not entirely according to spec but it prevents a precedence issue for fieldnames in brackets `{ [fieldname]: value }`. I found this when making PR #16 but solved by this change.